mmseg/datasets/pascal_context.py

```diff
@@ -46,18 +46,18 @@ class PascalContextDataset(BaseSegDataset):
                  [255, 71, 0], [0, 235, 255], [0, 173, 255], [31, 0, 255]])
 
     def __init__(self,
-                 ann_file: str,
+                 ann_file='',
                  img_suffix='.jpg',
                  seg_map_suffix='.png',
+                 reduce_zero_label=True,
```

Comment:
We can't set `reduce_zero_label` to `True` here.
https://mmsegmentation.readthedocs.io/en/latest/notes/faq.html#functionality-of-reduce-zero-label